irohad/main/server_runner.hpp

```diff
@@ -44,6 +48,23 @@ class ServerRunner {
    */
   void waitForServersReady();
 
+  /**
+   * Construct SSL credentials for a secure connection
+   * @return Result with server credentials or error message
+   */
+  iroha::expected::Result<std::shared_ptr<grpc::ServerCredentials>, std::string>
+  createSecureCredentials();
```

Comment:
There methods should definitely not be public. Please make them `private` or `protected`, or move them to an anonymous namespace in cpp file.